examples/dh-5-example/README.md

```diff
@@ -0,0 +1,27 @@
+This example helps Getting Started with using and configuring ingestion, mapping and mastering flows.
+There are 3 flow artifacts available in the flows directory.
+Before starting with this example, run the following gradle task to extract the input files directory from input.zip file
+./gradlew extractZip

```

Comment:
Need to run "hubInit extractZip" first, as no Gradle task will work unless hubInit is first run.

Reply:
Added it to the readme